### ISSUE TITLE
_handle_error not checking for missing error['value'] key

### ISSUE THREAD

**Issue by erikgreif-acc** [contentful/contentful-management.py#27]
I am trying to get started with this SDK and I ran into some trouble getting error feedback while learning how this all works. 

My code (yes, something is wrong with it, but hear me out):
```
import contentful_management
import calendar
import time

client = contentful_management.Client('REDACTED-KEY')
mainSpace = client.spaces().find('REDACTED-SPACE')

entry_attributes = {
    'content_type_id': 'open-api-yaml',
    'fields': {
        'title': {
            'en-US': "My Llama Taco"
        },
        'body': {
            'en-US': "test-body:\n  - Thing 1\n  - Thing 2\n  - Thing 3\n"
        },
        'timestamp': {
            'en-US': 0
        },
        'version_identifier': {
            'en-US': "1.0.0"
        }
    }
}

new_entry = mainSpace.entries().create('partitions-svc-openapi', entry_attributes)
```

When I run this, I should be getting back a useful error message, as the API behind this SDK is returning a 422. Problem is, I only see this:

```
Traceback (most recent call last):
  File "./testingContentful.py", line 37, in <module>
    new_entry = mainSpace.entries().create('partitions-svc-openapi', entry_attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/space_resource_proxy.py", line 48, in create
    return self.proxy.create(resource_id=resource_id, attributes=attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/entries_proxy.py", line 72, in create
    return super(EntriesProxy, self).create(resource_id=resource_id, attributes=attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/client_proxy.py", line 73, in create
    headers=self._resource_class.create_headers(attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/client.py", line 715, in _put
    return self._request('put', url, attributes, **kwargs)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/client.py", line 681, in _request
    error = get_error(response)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/errors.py", line 241, in get_error
    return error_class(response)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/errors.py", line 23, in __init__
    message = self._best_available_message(response)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/errors.py", line 57, in _best_available_message
    error_message.append("Details: {0}".format(self._handle_details(details)))
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/errors.py", line 160, in _handle_details
    errors.append(self._handle_error(error))
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/errors.py", line 153, in _handle_error
    error['value']
KeyError: 'value'
```

If I replace error['value'] with a constant string "TEMP", I receive this error as I would expect:

```
Traceback (most recent call last):
  File "./testingContentful.py", line 37, in <module>
    new_entry = mainSpace.entries().create('partitions-svc-openapi', entry_attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/space_resource_proxy.py", line 48, in create
    return self.proxy.create(resource_id=resource_id, attributes=attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/entries_proxy.py", line 72, in create
    return super(EntriesProxy, self).create(resource_id=resource_id, attributes=attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/client_proxy.py", line 73, in create
    headers=self._resource_class.create_headers(attributes)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/client.py", line 715, in _put
    return self._request('put', url, attributes, **kwargs)
  File "REDACTED/Python/3.6/lib/python/site-packages/contentful_management/client.py", line 683, in _request
    raise error
contentful_management.errors.UnprocessableEntityError: HTTP status code: 422
Message: No field with id "timestamp" found.
Details: 
	* Name: unknown - Path: '['fields', 'timestamp']' - Value: 'TEMP'
Request ID: REDACTED-REQUEST-ID
```

I believe something isn't assigning error['value'] like it is supposed to, but maybe that error handler simply needs to perform a check.

**Comment by Roblinde** [comment]
Thanks Erik!

Something is definitely wrong there, it seems like the management SDK makes an assumption about keys that aren't always present.

We will take care of it in the next release hopefully.

**Comment by dlitvakb** [comment]
Hey @erikgreif-acc,

Looks like I missed an edge case with error handling, I'll fix it and hopefully make a release today.

Cheers

**Comment by dlitvakb** [comment]
Fix released in version 1.5.2

Cheers
